db/migrate/20171018122656_add_image_url_to_gallery_images.rb

```diff
@@ -0,0 +1,32 @@
+# frozen_string_literal: true
+
+class AddImageUrlToGalleryImages < ActiveRecord::Migration
+  class GalleryImage < ActiveRecord::Base
+    validates :gallery_id, presence: true
+    validates :europeana_record_id,
+              presence: true, format: { with: Europeana::Record::ID_PATTERN }
+    validates :image_url, presence: true
+  end
+
+  def up
+    add_column :gallery_images, :image_url, :text
```

Comment:
I wanted to make explicit that this is the URL for the provider hosted image, but sure.
Be careful with your migrations if you've ran this locally already since I renamed it.